### ISSUE TITLE
[BUG] addon display control does not support folia

### ISSUE THREAD

**Issue by cgw2023** [Ghost-chu/QuickShop-Hikari#1786]
### Description

qsaddon discordsrv, residence and such can run in folia with `folia-supported: true` added in plugin.yml, but display control cannot run.

quickshop version is c9d4d6c6b0ebfcad808ecc8ab4cd5bebbcd4a5e8

### Steps to reproduce

1. install addon

### Expected Behaviour

addon can run

### Screenshots

![image](https://github.com/user-attachments/assets/8310d52d-768f-43cc-955b-77e344c02c9a)

### `/quickshop paste` URL

https://ghost-chu.github.io/quickshop-hikari-paste-viewer/?remote=https%3A%2F%2Fbytebin.lucko.me%2Fzcb5e4bFSs

### Checklist

- [ ] I'm running Paper or Spigot, and not a fork
- [ ] I confirm that Paper/Spigot has been updated to the latest build
- [X] I confirm that QuickShop-Hikari has been updated to the latest stable version released on Modrinth (or the latest CI version)
- [ ] I confirm that I have not read these checkboxes and therefore I just ticked them all.
- [X] I confirm that I'm using QuickShop-Hikari, not QuickShop-Reremake, and I'm well aware that they're maintained by different people, and that Reremake issues shouldn't be reported here.
- [X] I confirm that I am running a server that is not a Hybird Server, (e.g. Mohist, Magma, CatServer, Banner, etc.), and I am aware that QuickShop-Hikari may not function properly on a Forge/Fabric hybrid server, and I am running at my own risk on such a server program, and I am aware that the I run such server-side programs at my own risk and know that I will not receive any support or help for this behavior.
- [X] I am well aware that if the Issue Ticket is not filled out correctly and completely, it will simply be closed without any response or reason.

**Comment by creatorfromhell** [comment]
Add-ons are not supported for folia, and currently not planned to. This is why folia supported wasn't added to add-ons. This may change in the future,  but not until the core folia functionality is good to go.

**Comment by cgw2023** [comment]
> Add-ons are not supported for folia, and currently not planned to. This is why folia supported wasn't added to add-ons. This may change in the future, but not until the core folia functionality is good to go.

so for the problem #1683 I faced in folia, I should instead create another issue report? since it is not for folia and then there is clearly a bug for bedrock folia players

**Comment by creatorfromhell** [comment]
> > Add-ons are not supported for folia, and currently not planned to. This is why folia supported wasn't added to add-ons. This may change in the future, but not until the core folia functionality is good to go.
> 
> so for the problem #1683 I faced in folia, I should instead create another issue report? since it is not for folia and then there is clearly a bug for bedrock folia players

As stated here, the add-ons aren't currently planned for folia

**Comment by creatorfromhell** [comment]
Also the only supported display type that will be supported going forward is virtual displays.
